crates/lakekeeper/src/implementations/postgres/tabular/mod.rs

```diff
@@ -393,8 +396,7 @@ pub(crate) async fn create_tabular(
         r#"SELECT EXISTS (
                SELECT 1
                FROM tabular ta
-               JOIN namespace n ON ta.namespace_id = n.namespace_id
-               JOIN warehouse w ON w.warehouse_id = n.warehouse_id
+               JOIN warehouse w ON ta.warehouse_id = w.warehouse_id
                WHERE (fs_location = ANY($1) OR
                       -- TODO: revisit this after knowing performance impact, may need an index
                       (length($3) < length(fs_location) AND ((TRIM(TRAILING '/' FROM fs_location) || '/') LIKE $3 || '/%'))
```

Comment:
I couldn't find much on how a catalog has to handle `fs_location`, so I'm not sure if this query is still correct. Advice/pointers would be appreciated.